02-microarray/pathway-analysis_microarray_02_gsea.Rmd

```diff
@@ -278,18 +291,23 @@ The function we will use to map from Ensembl gene IDs to Entrez gene IDs is call
 Let's create a data frame that shows the mapped Entrez IDs along with the differential expression stats for the respective Ensembl IDs.
 
 ```{r}
-# First let's create a mapped data frame we can join to the differential expression stats
+# First let's create a mapped data frame we can join to the differential
```

Comment:
@jashapiro does this section need more info about `org.Dr.eg.db` per #435 or do you think it's sufficient to link to the more detailed identifier conversion example?

Reply:
Maybe a little note about where the mapIds() function comes from? Also, in those examples I moved `keytype` to right after `key` for "my brain works this way" reasons.

Also, is there a reason `entrez_id` is in quotes? I don't think we do that elsewhere.

Reply:
> Maybe a little note about where the mapIds() function comes from? Also, in those examples I moved `keytype` to right after `key` for "my brain works this way" reasons.

👍 
> Also, is there a reason `entrez_id` is in quotes? I don't think we do that elsewhere.

Not that I'm aware of, I can change that.